src/GraphQL/Types/GraphType.cs

```diff
@@ -83,7 +83,11 @@ public string Name
         public string Description { get; set; }
 
         /// <inheritdoc/>
-        public string DeprecationReason { get; set; }
+        public string? DeprecationReason
```

Comment:
NRT annotation in class without NRTs

Reply:
It inherits `string? IProvideDeprecationReason.DeprecationReason { get; set; }`

Reply:
Whatever you think. It will just produce compiler warnings for now

Reply:
If I continue to set NRT annotation, it will become avalanche-like process.